src/ansys/pytwin/evaluate/twin_model.py

```diff
@@ -235,68 +236,68 @@ def _instantiate_twin_model(self):
             self._initialize_outputs_with_none_values()
 
         except Exception as e:
-            msg = "Twin model failed during instantiation!"
+            msg = "Twin model failed during instantiation."
             msg += f"\n{str(e)}"
             self._raise_error(msg)
 
     def _raise_model_error(self, msg):
         """
-        Raise a TwinModelError with formatted message.
+        Raise a TwinModelError with a formatted message.
```

Comment:
```suggestion
        Raise a TwinModelError with a formatted message.
```
Tough one! Can it be "Raise a formatted ``TwinModelError`` message."

Reply:
done